### DISCUSSION TITLE
REDCap: Darstellung für 1a Routinebetrieb

### DISCUSSION THREAD

**Discussion by cDataPusher** [medizininformatik-initiative/INTERPOLAR#527] [General]
Zum Start der IP-1a-Dokumentation im Routinebetrieb wurden bestimmte Darstellungen in REDCap anders gewünscht. Diese Wünsche sowie mögliche Anpassungen möchte ich hier gerne mit allen diskutieren.

Uns ist Folgendes aufgefallen:

1. Fall-Formular: Im Fall-Formular werden die aktuellste Station und die aktuellste Zimmernummer dargestellt. Wenn Patient:innen auf eine Nicht-Interpolar-Station wechseln, sind sie nicht mehr über die Berichte auffindbar.
Vorschlag: Das Stations-Feld im Frontend könnte zur Darstellung der zuletzt bekannten IP-Station genutzt werden. Die Zimmernummer könnte zur Anzeige der aktuellen Zimmernummer mit Stationskennung verwendet werden, unabhängig davon, ob es sich um eine IP-Station handelt.
4. Stationsberichte für Apotheker:innen: Die Apotheker:innen haben sich einen Bericht für jede Station gewünscht, um die aktuell vorhandenen Patient:innen zu sehen (Station = IP-Station, Entlassdatum ist null).
6. Darstellung in Berichten: Zur besseren Übersichtlichkeit wurde eine Darstellung mit einer Zeile pro Patient:in gewünscht. Hierfür haben wir die REDCap-Extension "Extended Reports" hinzugefügt und zeigen dabei stets den letzten Stand an.
8. Darstellung der Diagnosen: In REDCap werden derzeit alle Aufnahmediagnosen dargestellt. Alternativ könnten wir alle Diagnosen anzeigen und dabei die nicht MRP-relevanten herausfiltern. Das Filtern von Informationen macht daraus nicht automatisch ein Medizinprodukt.

Fragen an die Runde: Welche weiteren Besonderheiten sind aufgefallen? Können die genannten Punkte bestätigt werden oder gibt es zusätzliche Anmerkungen?

**Comment by t-beppler** [comment]
Aktuelle Implementierung bzgl. Fall-Verlegung und Patienten-Wiederaufnahme:
- Verlegung innerhalb eines Falls von einer INTERPOLAR-Station auf eine NICHT-INTERPOLAR-Station (gleicher Fall): Fall bleibt im REDCap auf INTERPOLAR-Station, Zimmernummer ändert sich wie oben beschrieben auf die der NICHT-INTERPOLAR-Station
- Verlegung innerhalb eines Falls von einer INTERPOLAR-Station auf eine andere INTERPOLAR-Station (gleicher Fall): Station wird im REDCap aktualisiert, sodass die aktuellste INTERPOLAR-Station angezeigt wird. Die Data Access Group für den Record wird auf die zweite Station gesetzt und der Patient nur für diese Data Access Group (= Station) angezeigt.
- Entlassung (von einer INTERPOLAR-Station) und Aufnahme auf eine andere INTERPOLAR-Station (zwei verschiedene Fälle): Im REDCap ist der Patient nur für die Station bzw. Data Access Group des aktuellen Falls sichtbar.
